src/views/statements/transactionStatements.view.ts

```diff
@@ -0,0 +1,160 @@
+/*
+ *
+ * Copyright 2018-present NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import {
+ TransactionStatement,
+ Receipt,
+ ReceiptType,
+ BalanceTransferReceipt,
+ BalanceChangeReceipt,
+ InflationReceipt,
+ ArtifactExpiryReceipt,
+ MosaicId,
+} from 'symbol-sdk'
+import {AbstractStatementView} from './abstractStatementView.view'
+import {CellRecord} from '../transactions/details/transaction.view'
+import {RecipientsView} from '../recipients.view'
+
+export class TransactionStatementViews extends AbstractStatementView {
+ public constructor(private readonly statements: TransactionStatement[]) {super()}
+
+ /**
+  * Renders cell records
+  * @returns {CellRecord[]}
+  */
+ public render(): CellRecord[] {
+  if (!this.statements.length) {return []}
+
+  return [
+   this.getSectionTitle('Transaction statements'),
+   ...this.statements
+    .map(({height, source, receipts}, index, self) => ([
+     this.getSectionTitle(`Transaction statement ${index + 1} of ${self.length}`),
+     {height: height.compact()},
+     {source: `Primary Id: ${source.primaryId}, Secondary Id: ${source.secondaryId}`},
+     this.getSectionTitle('Transaction receipts'),
+     ...this.renderReceipts(receipts),
+    ]))
+    .reduce((acc, entry) => ([...acc, ...entry]), []),
+  ]
+ }
+
+ /**
+  * Renders receipts
+  * @private
+  * @param {Receipt[]} receipts
+  * @returns {CellRecord[]}
+  */
+ private renderReceipts(receipts: Receipt[]): CellRecord[] {
+  return receipts
+   .map((receipt, index, self) => this.renderReceipt(receipt, index, self.length))
+   .reduce((acc, entry) => ([...acc, ...entry]), [])
+ }
+
+ /**
+  * Renders a receipt
+  * @private
+  * @param {Receipt} receipt
+  * @param {number} index
+  * @param {number} numberOfReceipts
+  * @returns {CellRecord[]}
+  */
+ private renderReceipt(
+  receipt: Receipt,
+  index: number,
+  numberOfReceipts: number,
+ ): CellRecord[] {
+  const {type, size} = receipt
+  return [
+   this.getSectionTitle(`Receipt ${index + 1} of ${numberOfReceipts}`),
+   {type: ReceiptType[type]},
```

Comment:
``size`` and ``type`` first letters should be capitalized  to match other properties
![image](https://user-images.githubusercontent.com/9107969/77303125-4b517c80-6cea-11ea-9eb5-0d1eaf89a9ae.png)